x-pack/plugins/data_usage/server/services/autoops_api.ts

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import https from 'https';
+import { SslConfig, sslSchema } from '@kbn/server-http-tools';
+import apm from 'elastic-apm-node';
+
+import type { AxiosError, AxiosRequestConfig } from 'axios';
+import axios from 'axios';
+import { LogMeta } from '@kbn/core/server';
+import {
+  UsageMetricsRequestSchemaQueryParams,
```

Comment:
Note: Now that we made the API a POST request  `UsageMetricsRequestSchemaQueryParams` is going to be renamed to UsageMetricsRequestBody in /elastic/kibana/pull/195556/files